tests/baselines/reference/isolatedDeclarationErrorsClasses.errors.txt

```diff
@@ -52,13 +53,16 @@ isolatedDeclarationErrorsClasses.ts(56,5): error TS9013: Expression type can't b
     
         get getOnly() { return 1 + 1 }
             ~~~~~~~
-!!! error TS9009: At least one accessor must have an explicit return type annotation with --isolatedDeclarations.
+!!! error TS9009: At least one accessor must have an explicit type annotation with --isolatedDeclarations.
 !!! related TS9032 isolatedDeclarationErrorsClasses.ts:11:9: Add a return type to the get accessor declaration.
         set setOnly(value) { }
             ~~~~~~~
 !!! error TS7032: Property 'setOnly' implicitly has type 'any', because its set accessor lacks a parameter type annotation.
                     ~~~~~
 !!! error TS7006: Parameter 'value' implicitly has an 'any' type.
+                    ~~~~~
+!!! error TS9009: At least one accessor must have an explicit type annotation with --isolatedDeclarations.
+!!! related TS9033 isolatedDeclarationErrorsClasses.ts:12:9: Add a type to parameter of the set accessor declaration.

```

Comment:
Is there a way to silence this by chance? Usually we try to avoid duplicate errors but maybe it's fine...

Reply:
This is also an issue for parameters ([Playground Link](https://www.typescriptlang.org/play/?noImplicitAny=false&isolatedDeclarations=true#code/MYGwhgzhAEAa0G8BQ1ougWwBQAcCUAXNAG4D2AlgCaJLoC+SdQA)). I think we can suppress these errors if the user also has `noImplicitAny` turned on. But I think we should do that in a separate PR so we don't overload this PR with other changes.